nri/index.html

```diff
@@ -1 +1,299 @@
 <!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <link
+      href="https://use.fontawesome.com/releases/v5.6.1/css/all.css"
+      rel="stylesheet"
+    />
+    <meta name=”description” content=”野村総合研究所（NRI）の公式ホームページです。NRIからの提言や調査・レポート、商品サービス、ITソリューション事例、IR情報、採用情報、サステナビリティ情報などを掲載しています。”>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
+    <title>NRI課題</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="logo">NRI</h1>
+      <nav>
+        <ul class="main-nav-left">
+          <li><a href="#">ナレッジ・インサイト</a></li>
+          <li><a href="#">ソリューション・サービス</a></li>
+          <li><a href="#">サステナビリティ</a></li>
+          <li><a href="#">ニュース</a></li>
+          <li><a href="#">企業情報</a></li>
+          <li><a href="#">株主・投資家情報（IR）</a></li>
+          <li><a href="#">採用情報</a></li>
+        </ul>
+        <ul class="main-nav-right">
+          <li><i class="fas fa-search search fa-lg"></i></li>
+          <li><i class="fas fa-globe globe fa-lg"></i></li>
+          <li><i class="fas fa-align-justify line fa-lg"></i></li>
+        </ul>
+      </nav>
+    </header>
+
+<ul>
+  <li>NRI トップ</li>
+  <li>&gt;</li>
+  <li>企業情報</li>
+  <li>&gt;</li>
+  <li> NRIの強み：NRIのデジタルビジネス戦略</li>
+</ul>
```

Comment:
- [ ] ulのインデントを上の`header`や下の`div`と合わせてください
- [ ] `&gt;`は、ulの箇条書きの内容としてふさわしくないので移動させましょう

```suggestion
<ul>
  <li>NRI トップ&gt;</li>
  <li>企業情報&gt;</li>
  <li> NRIの強み：NRIのデジタルビジネス戦略</li>
</ul>
```